go/vt/vttablet/tabletserver/throttle/throttler_test.go

```diff
@@ -75,3 +116,93 @@ func TestIsAppExempted(t *testing.T) {
 	throttler.UnthrottleApp("schema-tracker") // meaningless. App is statically exempted
 	assert.True(t, throttler.IsAppExempted("schema-tracker"))
 }
+
```

Comment:
A short comment would still be nice here.

Reply:
Added.